### ISSUE TITLE
Way to know the current selected image

### ISSUE THREAD

**Issue by ludem** [andyyou/react-coverflow#81]
I'm wondering what's the way to know what image is current selected, i.e. image in the middle of the flow. Is it possible to add a method to handle selected image change?

Thank you

**Comment by KTrentLR** [comment]
I agree.  I am using this as a means to go through cards in a players hand, I would like to know what the card is they have in focus.
